src/server/middlewares/validar-campos.js

```diff
@@ -0,0 +1,11 @@
+import { validationResult } from 'express-validator';
+
+export const validarCampos = (req, res, next) => {
```

Comment:
I would put middlewares in the `middlewares` folder only if they are more generic. Thisone is pretty much specific to the validation of a form use case. I would rather have this directly on the handler of that specific action.

Reply:
Hmmmm I think this one is tied to the `check` of `express-validation`. right? it's not immediate to understand that.